integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/longstring/VarcharDataTypesBasicTestCase.scala

```diff
@@ -434,6 +437,24 @@ class VarcharDataTypesBasicTestCase extends QueryTest with BeforeAndAfterEach wi
 
     sql("DROP TABLE IF EXISTS varchar_complex_table")
   }
+
+  test("check new columns after modifying schema through alter table queries") {
+    sql(
+      s"""
+         | CREATE TABLE if not exists $longStringTable(
```

Comment:
as i can see, there are no long string columns, the table property `long_string_columns` doesnt have any value. SO how the unset long string affected. Basically we should avoid the flow going too far when there are no long string columns itself. can you check?

Reply:
Done